### PR TITLE
Updated new features in Readme with sample workflows

### PR DESCRIPTION
**Pull request by Josh-01**
Added workflow samples for using the new env variables to enable traceability features for dockerfile, manifests, helmcharts and acr.

### CODE REVIEW

**Comment by Josh-01** [comment]
V1.2 has been updated, so the new features are available in the action. I can now merge this PR once it is approved, so that users can see the new env-variables/ labels expected for traceability in the README.

**Review by koushdey — APPROVED** [review]
Check on the integration test failures. I think those are not because of you.

File: README.md
Comment (ds-ms) [inline README.md:332]: I don't think this is a good design. 

Also take this change along with the PR.

Comment (Josh-01) [reply README.md]: I agree it was bad design - I have reverted this particular change. We have already pushed the current changes to v1.2, so we should atleast update the README to reflect those changes - this is long overdue.
I will raise a new set of PRs with the design changes required and all other review comment changes, for master(with updated README), v1.2,v1.3 and v1.4.

Comment (ds-ms) [reply README.md]: Why did you merge this PR? Given that you were changing the logic anyway?

Comment (Josh-01) [reply README.md]: Like I mentioned in my above comment while merging, the changes had been discussed and pushed to v1.2 in code, and only the README update was pending for a long time. After discussing with Ambika, he suggested we go ahead with the merge and add the new changes as an incremental update, since it was pending discussion.

**Pull request merged**
